card.io/src/main/jni/nativeRecognizer.cpp

```diff
@@ -287,8 +296,22 @@ void setDetectedCardImage(JNIEnv* env, jobject jCardResultBitmap, IplImage* card
 
     IplImage* cardResult = cvCreateImageHeader(cvSize(bmInfo.width, bmInfo.height), IPL_DEPTH_8U, 4);
     cvSetData(cardResult, pixels, bmInfo.stride);
-
     dmz_YCbCr_to_RGB(cardY, bigCb, bigCr, &cardResult);
+
+    // blur cardnumber
```

Comment:
Typically we've tried to keep image processing code in the [DMZ repo](https://github.com/card-io/card.io-dmz) so that we can keep functional parity with iOS.